helmfile/values/gatekeeper/podsecuritypolicies.gotmpl

```diff
@@ -0,0 +1,62 @@
+constraints:
+  ingress-nginx:
+    nginx:
+      labelsSelector:
+          app.kubernetes.io/component: controller
+      allow:
+        allowedCapabilities:
+          - NET_BIND_SERVICE
+        {{- if .Values.ingressNginx.controller.useHostPort }}
+        hostPort: true
+        {{- end }}
+        hostNamespace: {}
+        privilege: {}
```

Comment:
I'm thinking just for style that we should set `true` even though it's enough that the key exists.